src/main/java/com/jalasoft/jfc/model/Param.java

```diff
@@ -28,6 +28,17 @@
   //Name the file that will be converted.
   private String outputFileName;
 
+  //It is a Md5 code of a file
+  private String md5;
+
+  /**
+   * getMd5 gets the checksum value.
```

Comment:
the name of method is not necesary **getMd5**, you can use **This methos gets ...**